src/generic/datavgen.cpp

```diff
@@ -2,7 +2,8 @@
 // Name:        src/generic/datavgen.cpp
 // Purpose:     wxDataViewCtrl generic implementation
 // Author:      Robert Roebling
-// Modified by: Francesco Montorsi, Guru Kathiresan, Bo Yang
+// Modified by: Francesco Montorsi, Guru Kathiresan, Bo Yang,
+//              Konstantin S. Matveyev
```

Comment:
This is minor, but please don't do this, this is pre-Git practice and we should really just get rid of all these "Modified by" lines. With Git it's simple to just have a look at `git shortlog` to see more accurate and always up-to-date information.

And we also maintain change log with contributor credits separately, too.

Reply:
I'll revert this.
It's hard to do everything as expected (in 2020) while you are working with the source from 1998 :^)

Reply:
We're moving it forwards, one step at a time...